target_chains/ethereum/contracts/test/pyth.js

```diff
@@ -1069,6 +1075,88 @@ contract("Pyth", function () {
     // and adding it here will cause more complexity (and is not so short).
   });
 
+  it("Setting wormhole address should work", async function () {
+    // Deploy a new wormhole contract
+    const newSetup = await Setup.new();
+    const newImpl = await Implementation.new();
+
+    // encode initialisation data
+    const initData = newSetup.contract.methods
+      .setup(
+        newImpl.address,
+        [testSigner1.address],
+        governance.CHAINS.polygon, // changing the chain id to polygon
+        wormholeGovernanceChainId,
+        wormholeGovernanceContract
+      )
+      .encodeABI();
+
+    const newWormhole = await Wormhole.new(newSetup.address, initData);
```

Comment:
should we test this using `WormholeReceiver`? (or add a separate test case for it)

Reply:
good point, yes! i'll do.